cmd/clusterctl/examples/azure/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure Example Files
+
+## Contents
+
+*.yaml files - concrete example files that can be used as is.
+*.yaml.template files - template example files that need values filled in before use.
+
+## Generation
+
+For convenience, a generation script which populates templates based on azure configuration is provided.
+
+1. Run the generation script.
+
+``` shell
+./generate-yaml.sh
+```
+
+<<<<<<< HEAD
```

Comment:
Unresolved merge conflict?

Reply:
Weird this didn't show on my local. Will fix.